dpgen/generator/run.py

```diff
@@ -1626,17 +1626,22 @@ def check_bad_box(conf_name,
             raise RuntimeError('unknow key', key)
     return is_bad
 
-
 def _read_model_devi_file(
         task_path : str,
-        model_devi_f_avg_relative : bool = False
+        model_devi_f_avg_relative : bool = False,
+        model_devi_merge_traj : bool = False
 ):
     model_devi = np.loadtxt(os.path.join(task_path, 'model_devi.out'))
     if model_devi_f_avg_relative :
-        trajs = glob.glob(os.path.join(task_path, 'traj', '*.lammpstrj'))
-        all_f = []
-        for ii in trajs:
-            all_f.append(get_dumped_forces(ii))
+        if(model_devi_merge_traj is True) : 
+            all_traj = glob.glob(os.path.join(task_path, 'all.lammpstrj'))
+            all_f = get_all_dumped_forces(all_traj)
```

Comment:
It looks that `get_all_dumped_forces` accepts a `str`, but `all_traj` is a `list`?